SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs

```diff
@@ -300,23 +300,60 @@ public static partial class CsmManager
             }
         }
 
-        private static async Task<Site> CreateSite(ResourceGroup resourceGroup, Func<string> nameGenerator)
+        private static async Task<Site> CreateSite(ResourceGroup resourceGroup, Func<string> nameGenerator, string siteKind = null)
+        {
+            var site = new Site(resourceGroup.SubscriptionId, resourceGroup.ResourceGroupName, nameGenerator(), siteKind);
+            site.Kind = siteKind;
+            await resourceGroup.LoadServerFarms(null);
```

Comment:
The default value on the parameter is a `null` . I can check if passing nothing translates to the same thing.